addons/storyshots/src/test-bodies.js

```diff
@@ -39,3 +43,63 @@ export function renderOnly({ story, context }) {
   const storyElement = story.render(context);
   reactTestRenderer.create(storyElement);
 }
+
+export const imageSnapshotWithOptions = ({ port = 6006, host = 'localhost', scheme = 'http' }) => {
+  let browser; // holds ref to browser. (ie. Chrome)
+  let page; // Hold ref to the page to screenshot.
+
+  const testFn = ({ context }) => {
+    const baseUrl = `${scheme}://${host}:${port}`;
+    const encodedKind = encodeURIComponent(context.kind);
+    const encodedStoryName = encodeURIComponent(context.story);
+    const storyUrl = `/iframe.html?selectedKind=${encodedKind}&selectedStory=${encodedStoryName}`;
```

Comment:
As I understand `imageSnapshot` won't work for RN. It makes this addon a bit inconsistent. This should be documented very clearly. You can even throw some error in case someone tries to run it for RN.

Reply:
I added a message about it inside README & the test itself.